src/main/java/org/tdl/vireo/service/SystemDataLoader.java

```diff
@@ -411,30 +411,22 @@ private void loadOrganization() {
                 organization = organizationRepo.create(systemOrganization.getName(), category);
                 organization.setAcceptsSubmissions(systemOrganization.getAcceptsSubmissions());
             }
-            // else set systemOrganization to existing organization
-            else {
-                systemOrganization = organization;
```

Comment:
System organization is a non-entity managed object which is object mapped from the JSON file `/organization/SYSTEM_Organization_Definition.json`. Reassigning to the entity managed organization is not necessary.